test/Project.toml

```
@@ -38,10 +38,10 @@ AdvancedPS = "0.3"
 AdvancedVI = "0.1"
 Clustering = "0.14"
 CmdStan = "6.0.8"
-Distributions = "0.25"
+Distributions = "<0.25.65"
```

Due to the deprecation issues in 0.25.65, I assume? They are fixed in 0.25.66 so I guess this can be reverted:

```suggestion
Distributions = "0.25"
```